biorbd_optim/optimal_control_program.py

```diff
@@ -238,7 +238,13 @@ def __define_multiple_shooting_nodes_per_phase(self, nlp, idx_phase):
             else:
                 V_bounds.min[offset : offset + nlp["nx"]] = nlp["X_bounds"].min
                 V_bounds.max[offset : offset + nlp["nx"]] = nlp["X_bounds"].max
-            V_init.init[offset : offset + nlp["nx"]] = nlp["X_init"].init
+            if nlp["X_init"].init_lineaire:
+                V_init.init[offset : offset + nlp["nx"]] = [
+                    (nlp["X_init"].init[1 + 2*i] - nlp["X_init"].init[2*i]) * (k/nlp["ns"]) +
+                    nlp["X_init"].init[2*i] for i in range(nlp["nx"])
+                ]
+            else:
+                V_init.init[offset: offset + nlp["nx"]] = nlp["X_init"].init
```

Comment:
This computation doesn't belong to optimal_control_program, but the path_condition. There should have a get_init function (or something similar) that returns the proper values depending on the type of initial that is asked

Reply:
Thank you for your answer, and the two other following.
I take it into account and will modify it. `get_init` is a good name for the function.